packages/botonic-react/src/webchat/webchat.jsx

```diff
@@ -286,16 +285,16 @@ export const Webchat = forwardRef((props, ref) => {
   }
 
   const handleMenu = () => {
-    setEmojiPickerIsOpened(false)
-    persistentMenuIsOpened
-      ? setPersistentMenuIsOpened(false)
-      : setPersistentMenuIsOpened(true)
+    toggleEmojiPicker(false)
+    webchatState.isPersistentMenuOpen
```

Comment:
personally I prefer:
togglePersistentMenu(!webchatState.isPersistentMenuOpen)
the "!" makes it clear that we're toggling
same for handleEmojiClick